classes/MockDelegateFunctionBuilder.php

```diff
@@ -52,11 +52,13 @@ public function build($functionName = null)
         $parameterBuilder->build($functionName === null ? '' : $functionName);
         $signatureParameters = $parameterBuilder->getSignatureParameters();
 
+        $populatedFunctionName = $functionName === null ? '_dummy' : $functionName;
```

Comment:
@crocodele actually I am thinking if we should even create a method in the class, when the function name is null?

I am not really sure why we even allow the function name to be null here, and it looks like the null value is not even used in the satellite packages.

Also I wonder if we should be creating the method with a `signatureParameters`? As I see all seems to be working fine without the 'correct signature' tho 🤔